migrations/1623716682037-DropRfqmStatusReasonColumn.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DropRfqmStatusReasonColumn1623716682037 implements MigrationInterface {
+    name = 'DropRfqmStatusReasonColumn1623716682037';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "rfqm_jobs" DROP COLUMN "status_reason"`);
```

Comment:
small nit, other migrations use the typeorm classes (e.g. Column class) instead of raw SQL, but I don't have a strong preference